cmd/renterd/main.go

```diff
@@ -125,7 +125,9 @@ func mustLoadAPIPassword() {
 		return
 	}
 
-	fmt.Print("Enter API password: ")
+	fmt.Println("Please choose a password to unlock the UI and API.")
+	fmt.Println("(The password must be at least 4 characters.)")
```

Comment:
If we're not asserting these password criteria we shouldn't be printing them I think. I would personally find it a little annoying to have password requirements but at the same time it's probably a good idea, in which case we should use more meaningful criteria.

Reply:
@peterjan Can you please confirm if we should keep this criteria or can you provide the criteria details that should be updated here ?

Reply:
I would drop it for now. I think it's fine to introduce password criteria but perhaps that can be a separate PR. At the very least if we print the requirement we should assert the length of the password somewhere.